requirements.json

```diff
@@ -8,6 +8,7 @@
         "cups-dev",
         "eudev-dev",
         "ffmpeg-dev",
+        "g++",
```

Comment:
`g++` is already part of `build-base`, see:
<https://pkgs.alpinelinux.org/package/v3.13/main/x86_64/build-base>

Reply:
Oh, that's great!